### DISCUSSION TITLE
Getting Started with Open Source Contributions at CNCF

### DISCUSSION THREAD

**Discussion by Adityyyaaa** [cncf/mentoring#1312] [Q&A] [answered]
I, Aditya Kulkarni, want to express a strong interest in starting Open Source Contributions @CNCF. With a strong academic background in Mathematics and Computing from the Indian Institute of Technology (BHU), Varanasi, graduating in May 2025, and extensive experience in software development and cloud technologies, I am confident I can contribute effectively to your team.

I have interned at Schrodinger, Inc. (D. E. Shaw Group), wherein I worked on LiveDesign Application deployment on AWS, increasing the throughput for persistent volumes by 6.5 times. I further had to optimise the throughput of RWX-based pods, which eventually had 6.3 times more throughput with excellent backing-up capabilities. I have had significant exposure to technologies like Kubernetes, Docker, Amazon Web Services, Google Cloud Platform, and Microsoft Azure.

I have extensively worked on full-stack development projects, which include a platform for women's community to resolve their everyday issues and build a solid community for women to discuss sensitive issues and get required support regarding problems such as menstrual gaps, mental well-being and other sensitive issues. Further, I have built a customised command line of UNIX type, where I can give in-built commands and preprocess them.

I am excited about the opportunity to contribute to your cutting-edge projects and collaborate with some of the brightest minds and contribute to open source in the best possible way I can.

**Comment by nate-double-u** [comment, marked as answer]
The best place to get started is over in our [Slack workspace](https://communityinviter.com/apps/cloud-native/cncf), check out the pinned links in the `#cncf-new-contributors` channel there.

**Comment by FARDEEN-785** [comment]
Hello,
I’m Mohd Fardeen Khan, a BTech student in Computer Science. I have a good understanding of Python and Flask, and I’m really interested in learning and contributing to the cloud-native ecosystem.

CNCF’s work in advancing cloud-native computing really inspires me, and I want to get involved with open-source projects in this space. I’d love some guidance on:

Beginner-friendly projects or tasks to start with.
Areas where contributors are currently needed.
Resources to better understand CNCF workflows and projects
.
I’m eager to learn, grow, and contribute to this amazing community. Looking forward to your suggestions!

with regards
Mohd Fardeen khan
